klaviyo/public.py

```diff
@@ -99,7 +100,7 @@ def track_once(
         return self.track(event, email=email, external_id=external_id, properties=properties, customer_properties=customer_properties,
             timestamp=timestamp, ip_address=ip_address, is_test=is_test)
 
-    def identify(self, email=None, external_id=None, properties={}, is_test=False):
+    def identify(self, email=None, external_id=None, properties={}, is_test=False, method=KlaviyoAPI.HTTP_GET):
```

Comment:
Related to my comment on the thread above.

the default value for the properties kwarg should be None and set to an empty dict in the method logic (already handled actually).

Example printing out the properties variable after modifying it with email/id
```
In [5]: client.Public.identify(external_id='12345')
{'id': '12345'}

In [6]: client.Public.identify('scott@klaviyo.com')
{'id': '12345', 'email': 'scott@klaviyo.com'}
```

versus is the default value is None
```
In [3]: client.Public.identify(external_id='12345')
{'id': '12345'}

In [4]: client.Public.identify('scott@klaviyo.com')
{'email': 'scott@klaviyo.com'}
```

Reply:
This was old code but Rems comment made me think of this. If you could update that it'd be awesome!

Reply:
lets do that after this push as a followup, we need to get this PR out